hunts/src/AnswerCell.js

```diff
@@ -1,31 +1,103 @@
+import Badge from "react-bootstrap/Badge";
 import Button from "react-bootstrap/Button";
 import React from "react";
 import { updatePuzzle } from "./puzzlesSlice";
 import { useSelector, useDispatch } from "react-redux";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faEdit, faTrashAlt } from "@fortawesome/free-regular-svg-icons";
+import { faPlus } from "@fortawesome/free-solid-svg-icons";
 import { showModal } from "./modalSlice";
 
 export default function AnswerCell({ row, value }) {
   const { id: huntId } = useSelector((state) => state.hunt);
   const dispatch = useDispatch();
-  return value ? (
-    <span className="text-monospace">{value}</span>
-  ) : (
-    <Button
-      variant="outline-primary"
-      onClick={() =>
-        dispatch(
-          showModal({
-            type: "SUBMIT_ANSWER",
-            props: {
-              huntId,
-              puzzleId: row.values.id,
-              puzzleName: row.values.name,
-            },
-          })
-        )
-      }
-    >
-      Submit Answer
-    </Button>
+  if (row.original.guesses === undefined || row.original.guesses.length == 0) {
+    return (
+      <Button
+        variant="outline-primary"
+        onClick={() =>
+          dispatch(
+            showModal({
+              type: "SUBMIT_ANSWER",
+              props: {
+                huntId,
+                puzzleId: row.values.id,
+                puzzleName: row.values.name,
+              },
+            })
+          )
+        }
+      >
+        Submit Answer
+      </Button>
+    );
+  }
+  return (
+    <>
+      {row.original.guesses.map(({ id, text }) => (
```

Comment:
The children from this `.map` need a `key` prop-- there's a warning message in the dev console about it.

Reply:
I couldn't figure out how to make this disappear -- I added keys to every fragment and span inside this map.